### PR TITLE
Use range loop

### PR DESCRIPTION
**Pull request by ShFil119**
Small clean up. It should increases readability a bit.  
- [x] Some of values are hardcoded, I mean written by a hand. Not sure, is it a part of gta3? 
- [x] some of them can be const
- [x] madebr's suggestions

And most important - has this pull a meaning or value? Do you want range loops?

### CODE REVIEW

File: rwgame/MenuSystem.hpp [hunk 1]
```diff
@@ -126,9 +126,9 @@ class Menu {
 
     void click(const float x, const float y) {
         glm::vec2 c(x - offset.x, y - offset.y);
-        for (auto it = entries.begin(); it != entries.end(); ++it) {
+        for (auto &entrie : entries) {
```
Comment (madebr) [inline rwgame/MenuSystem.hpp]: entry

File: rwlib/source/loaders/LoaderIMG.cpp [hunk 2]
```diff
@@ -40,9 +40,9 @@ bool LoaderIMG::load(const rwfs::path& filepath) {
 /// Get the information of a asset in the examining archive
 bool LoaderIMG::findAssetInfo(const std::string& assetname,
                               LoaderIMGFile& out) {
-    for (size_t i = 0; i < m_assets.size(); ++i) {
-        if (boost::iequals(m_assets[i].name, assetname)) {
-            out = m_assets[i];
+    for (auto &m_asset : m_assets) {
```
Comment (madebr) [inline rwlib/source/loaders/LoaderIMG.cpp]: m_asset -> asset

File: rwengine/src/data/GameTexts.cpp [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 GameString GameStringUtil::fromString(const std::string& str) {
     GameString s;
-    for (std::string::size_type i = 0u; i < str.size(); ++i) {
-        s += str[i];
+    for (const char &i : str) {
+        s += i;
```
Comment (madebr) [inline rwengine/src/data/GameTexts.cpp]: Is it possible to use `std::accumulate`?

Comment (ShFil119) [reply rwengine/src/data/GameTexts.cpp]: yeah, but code becomes ugly(lambda)

**Pull request merged**